cypress/features/regression/accessibility/a11y-third-part.test.js

```diff
@@ -0,0 +1,3 @@
+import tests from "../../../support/step-definitions/accessibility/a11y-utils";
+
+tests(601, 1000);
```

Comment:
This should be left open-ended to capture any new tests that go beyond the 1000 range.
[Slice does not include start](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) so it should start at `600`.
```suggestion
tests(600);
```